modules/gltf/gltf_document.cpp

```diff
@@ -6113,11 +6178,13 @@ void GLTFDocument::_convert_grid_map_to_gltf(GridMap *p_grid_map, GLTFNodeIndex
 		gltf_mesh.instantiate();
 		gltf_mesh->set_mesh(_mesh_to_importer_mesh(p_grid_map->get_mesh_library()->get_item_mesh(cell)));
 		gltf_mesh->set_original_name(p_grid_map->get_mesh_library()->get_item_name(cell));
+		String unique_name = _gen_unique_name(p_state, p_grid_map->get_mesh_library()->get_item_name(cell));
+		gltf_mesh->set_name(unique_name);
```

Comment:
Is the mesh and and the gltf node supposed to be set to the same unique_name?

Reply:
Yes, that's what we want. There's no conflicts if the mesh and node are named the same thing, and it would be confusing and harder to compare to the original scene to have the node be named something like `Cube` and the mesh named `Cube.001`.